common/remote_util.cc

```diff
@@ -123,9 +125,9 @@ absl::Status RemoteUtil::Sync(std::vector<std::string> source_filepaths,
 
   ProcessStartInfo start_info;
   start_info.command = absl::StrFormat(
-      "%s --ip=%s --port=%i -z %s %s%s",
-      path::Join(sdk_util_.GetDevBinPath(), "cdc_rsync"),
-      QuoteArgument(gamelet_ip_), ssh_port_,
+      "cdc_rsync.exe --ip=%s --port=%i -z "
```

Comment:
Can you drop the `.exe` extension? Windows doesn't need it and it's one less change to make if we want to port it to Linux.

Reply:
Done. I'm actually planning to remove this code since it's only used to deploy FUSE in debug mode, which was mainly used by Lena as her home internet wasn't so good. For us scp is probably even faster.